flexible_event_config.md

```diff
@@ -195,10 +200,10 @@ This example configuration supports a developer who wants to optimize for value
   {
     "trigger_data": [0],
     "event_report_windows": {
-      "end_times": [<7 days>]
+      "end_times": [<7 days>, <14 days>]
     },
     "summary_window_operator": "value_sum",
-    "summary_buckets": [5, 10, 100]
+    "summary_buckets": [1, 5, 10, 100]
```

Comment:
Why add the "1" here? Would be nice to have one example where the buckets don't start at 1

Reply:
Let me remove it. At some point I had a good reason to add this but after I added the other triggers I don't anymore.